custom_mutators/rust/README.md

```diff
@@ -5,7 +5,15 @@ Bindings to create custom mutators in Rust.
 These bindings are documented with rustdoc. To view the documentation run
 ```cargo doc -p custom_mutator --open```.
 
-A minimal example can be found in `example`. Build it using `cargo build --example example_mutator`. 
+A minimal example can be found in `example`. Build it using `cargo build --example example_mutator`.
 
 An example using [lain](https://github.com/microsoft/lain) for structured fuzzing can be found in `example_lain`.
 Since lain requires a nightly rust toolchain, you need to set one up before you can play with it.
+
+An example for the use of the post_process function, using [lain](https://github.com/microsoft/lain) with [serde](https://github.com/serde-rs/serde) and [bincode](https://github.com/bincode-org/bincode) can be found in `example_lain_post_process`.
```

Comment:
I would personally take a look use flexbuffers instead of bincode, I assume it should be faster

Reply:
(Not benchmarked that myself, though)

Reply:
Thanks I didn't know about this one, I'll look into it some day :)
So far out of all the solutions I tested I only managed to make bincode work. Lain's serialization function lose some information so my workaround was to serialize with bincode in the fuzz function, deserialize it in post_process to serialize it again in the right way for the target (and use AFL_POST_PROCESS_KEEP_ORIGINAL)...
So the performances are probably already terrible but not an issue compared to emulation with unicorn.